build/lib/release.sh

```diff
@@ -375,7 +375,7 @@ function kube::release::create_docker_images_for_server() {
         ln "${KUBE_ROOT}/build/nsswitch.conf" "${docker_build_path}/nsswitch.conf"
         chmod 0644 "${docker_build_path}/nsswitch.conf"
         cat <<EOF > "${docker_file_path}"
-FROM ${base_image}
+FROM --platform=linux/${arch} ${base_image}
```

Comment:
is the OS linux for all images produced?

Reply:
@liggitt -- Right. This is only for the server images, which we only release for Linux.